WordPressKit/AccountServiceRemoteREST.m

```diff
@@ -1,6 +1,4 @@
 #import "AccountServiceRemoteREST.h"
-#import "RemoteBlog.h"
-#import "RemoteBlogOptionsHelper.h"
```

Comment:
I intentionally started with the easy ones to translate. This `AccountServiceRemoteREST` falls into my harder basket, and will be dealt with after I have finished with the easy ones 🥲

Reply:
Sure.

Just to be clear, I was suggesting converting only that method, but I can see how adding a dedicated extension just for it would be wasteful given we'd remove it once the whole class is converted soon enough.